dags/bq_events_to_amplitude.py

```diff
@@ -150,3 +150,29 @@
         ),
         task_id=onboarding_task_id
     )
+
+    shredder_amplitude_fxa = gke_command(
```

Comment:
cc @relud that I'm moving this fxa amplitude shredder task into the bq_events_to_amplitude DAG.

Reply:
lgtm